src/reader/read_client.ts

```diff
@@ -197,6 +197,8 @@ export class ReadClient {
   }
 
   close() {
-    this._client.close();
+    this._client.close().catch(err => {
```

Comment:
why this change is needed for the upgrade ?

Reply:
Great question! It's from this generator update https://github.com/googleapis/gapic-generator-typescript/pull/1697 - we changed our lint rules and don't allow floating promises anymore, so you'll see more catch blocks like these in all of our code